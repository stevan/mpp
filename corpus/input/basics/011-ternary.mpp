$x > 5 ? "big" : "small";