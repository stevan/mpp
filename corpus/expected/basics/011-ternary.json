[
  {
    "type": "Ternary",
    "condition": {
      "type": "BinaryOp",
      "operator": ">",
      "left": {
        "type": "Variable",
        "name": "$x"
      },
      "right": {
        "type": "Number",
        "value": "5"
      }
    },
    "trueExpr": {
      "type": "String",
      "value": "\"big\""
    },
    "falseExpr": {
      "type": "String",
      "value": "\"small\""
    }
  }
]